dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -4525,6 +4525,28 @@ paths:
             $ref: "#/definitions/User"
       security:
       - BEARER: []
+  /users/{userId}/appTools:
```

Comment:
how does services work right now, is it also a separate endpoint?

Reply:
yep, there's a userServices endpoint